planning/behavior_velocity_planner/include/scene_module/occlusion_spot/occlusion_spot_utils.hpp

```diff
@@ -78,6 +78,8 @@ struct Velocity
   double max_stop_jerk;         // [m/s^3] emergency braking system jerk
   double max_stop_accel;        // [m/s^2] emergency braking system deceleration
   double max_slow_down_accel;   // [m/s^2] maximum allowed deceleration
+  double non_effective_jerk;    // [m/s^3] too weak jerk for velocity planning.
```

Comment:
`too` is misleading. Simply, this is no problem with `weak jerk for velocity planning.`

Reply:
fixed at 
[bfea98d](https://github.com/autowarefoundation/autoware.universe/pull/354/commits/bfea98d867805564378c02de02bfe78b6ffdbaba)